### ISSUE TITLE
Is video.js still needed in dspace/modules/?

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#614]
DSpace 5 contains but DSpace 6 does not contain dspace/modules/xmlui-mirage2/src/main/webapp/themes/vtmirage2/vendor/videojs/video.js # used for video playback, c532205. @keithgee, the videos seems to play. Do we need this file?

**Comment by keithgee** [comment]
No, it's no longer needed.

Videojs was used primarily to support some video playback features in Internet Explorer 6. Internet Explorer 6 is rarely seen anymore, so now we can rely on the HTML5 video playback features built into most web browsers.

I'd planned to remove all videojs references for when our theme was updated to the new look. It seems that may be a while, so I've cleaned it up now with commit dd750f491165cd9e5d74356a67c37a95800f3088.

I noticed an incorrect setting in our new configuration file that interferes with video playback; I'll address that in a separate commit.

***Closing
